README-wsl2-windows.md

```diff
@@ -0,0 +1,118 @@
+# Using WSL2 for esp-idf-template on Windows
+
+This doc will help you set up and use WSL2 (Windows Subsystem for Linux 2) for development using the esp-idf-template on Windows. 
+This method avoids [path length issues]([Error: Too long output directory · Issue #252 · esp-rs/esp-idf-sys (github.com)](https://github.com/esp-rs/esp-idf-sys/issues/252)) and provides a smoother development experience.
+
+## System Requirements
```

Comment:
I would replace all of "System Requirements", "Basic Prerequisites", and "Setup Steps" up to and including Step 4 with a simple link to a reliable source on the internet which contains all of that information, as I hope installing WSL2 itself  is in the meantime (a) common knowledge (b) not that complex.